sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_administration_client.hpp

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "azure/attestation/attestation_client_models.hpp"
+#include "azure/attestation/attestation_client_options.hpp"
+#include <azure/core/context.hpp>
+#include <azure/core/url.hpp>
+#include <string>
+
+namespace Azure { namespace Core { namespace Http { namespace _internal {
+  class HttpPipeline;
+}}}} // namespace Azure::Core::Http::_internal
+
+namespace Azure { namespace Security { namespace Attestation {
+
+  /**
+   *
+   * The AttestationAdministrationClient implements the functionality required by the
+   * "Administration" family of attestation service APIs.
+   *
+   * @note: Attestation administration APIs cannot be used on shared attestation service instances.
+   *
+   * The Administration family of APIs provide APIs to manage:
+   *
+   * - Attestation policies.
+   * - Attestation policy management certificates (Isolated attestation service instances only).
+   *
+   * There are three flavors of attestation service instances:
+   * -# Shared Mode
+   * -# AAD Mode
+   * -# Isolated Mode
+   *
+   * Shared mode attestation service instances do not allow any administration actions at all. They
+   * exist to allow customers to perform attestation operations without requiring any
+   * customizations.
+   *
+   * AAD Mode instances allow customers to modify attestation policies. When the attestation
+   * instance is in AAD mode, the creator of the instance indicates that they trust ARM RBAC and
+   * Microsoft AAD to validate client connections to the service. As such, additional authentication
+   * is not required for administrative operations.
+   *
+   *
+   */
+
+  class AttestationAdministrationClient final {
+
+  public:
+    /**
+     * @brief Destructor.
+     *
+     */
+    virtual ~AttestationAdministrationClient() = default;
+
+    /**
+     * @brief Construct a new Attestation Administration Client object
```

Comment:
nit: complete doc sentence with period
```suggestion
     * @brief Construct a new Attestation Administration Client object.
```